src/nemo-window.c

```diff
@@ -1166,6 +1167,13 @@ nemo_window_key_press_event (GtkWidget *widget,
 	active_slot = nemo_window_get_active_slot (window);
 	view = active_slot->content_view;
 
+      /**
+       * Disable the GTK Emoji Chooser
+       */
+      if ((event->keyval == GDK_KEY_semicolon || event->keyval == GDK_KEY_period) && (GDK_CONTROL_MASK + event->state == GTK_EMOJI_CHOOSER)) {
```

Comment:
event->state is a bitfield, so I think the second part of this condition should simply be:
```
(event->state & GDK_CONTROL_MASK)
```
Which checks is the control is being held, regardless of the state of other modifiers. This current patch doesn't work if you have capslock on or numlock off.

https://developer.gnome.org/gdk3/stable/gdk3-Windows.html#GdkModifierType

Reply:
Cleaned up the second part of the conditional.